### ISSUE TITLE
Disconnected relu layer

### ISSUE THREAD

**Issue by henriquetmaia** [dandelionmane/tf-dev-summit-tensorboard-tutorial#8]
Minor fix/suggestion, not sure if intentional (to make use view and use the graph)
but when we 'use_two_fc' then shouldn't the logits layer below use the relu output 'relu'?

`if use_two_fc:`
`    fc1 = fc_layer(flattened, 7 * 7 * 64, 1024, "fc1")`
`    relu = tf.nn.relu(fc1)`
...
`    logits = fc_layer(fc1, 1024, 10, "fc2")`

Thanks again!
